MultiSource/Benchmarks/Olden/power/main.c

```diff
@@ -61,12 +61,13 @@ _Unchecked int main(int argc, _Array_ptr<char*> argv : count(argc))
   double d_theta_R,d_theta_I;
```

Comment:
You could make the body of main a checked  scope by placing _Checked before the body of the function:
```
_Unchecked int main(int argc, ...) 
_Checked {
 ...
}
```